pkg/csi/service/vanilla/controller.go

```diff
@@ -189,7 +201,7 @@ func (c *controller) Init(config *config.Config) error {
 	}
 	// deletedVolumes timedmap with clean up interval of 1 minute to remove expired entries
 	deletedVolumes = timedmap.New(1 * time.Minute)
-	if config.FeatureStates.CSIMigration {
+	if containerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIMigration) {
 		common.CSIMigrationFeatureEnabled = true
```

Comment:
If `common.CSIMigrationFeatureEnabled` not used anywhere we should remove it.

If `common.CSIMigrationFeatureEnabled` updated automatically upon feature switch update in the config-map, we can use it all places instead of calling `containerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIMigration)`.

Reply:
common.CSIMigrationFeatureEnabled is used in full sync and utils. We can change it the next PRs